CMakeLists.txt

```diff
@@ -329,10 +329,13 @@ add_dependencies(sipi icc_profiles)
 
 
 # MESSAGE(STATUS "LIBS: " ${LIBS})
-target_link_libraries(sipi ${LIBS} lcms2 exiv2 expat jpeg tiff jbigkit png kdu_aux kdu xz magic lua jansson sqlite3 dl pthread curl ${CMAKE_DL_LIBS} z m )
+target_link_libraries(sipi ${LIBS} lcms2 exiv2 expat jpeg tiff jbigkit png kdu_aux kdu xz magic lua jansson sqlite3 dl pthread curl ${CMAKE_DL_LIBS} z m)
 if(CMAKE_SYSTEM_NAME STREQUAL DARWIN)
 	target_link_libraries(sipi iconv)
+else()
+    target_link_libraries(sipi rtmp rt)
```

Comment:
The library `librtmp` doesn't seem to be available as a binary package on CentOS. This is preventing Sipi from compiling on that platform. I've tried to get CMake to download `librtmp` and compile it, but without success, because `librtmp` seems to require OpenSSL (which Sipi doesn't), and in any case `librtmp` doesn't find the OpenSSL headers even if they're installed. Not sure what to do about this. Why do we need this library?